data/json/items/tool/toileteries.json

```
@@ -35,6 +35,31 @@
     "flags": [ "BELT_CLIP" ],
     "use_action": [ "WASH_HARD_ITEMS" ]
   },
+  {
+    "id": "kitchen_towel",
+    "type": "TOOL",
+    "category": "spare_parts",
+    "name": { "str": "kitchen towel" },
+    "description": "This is a kitchen towel, useful for cleaning hard surfaces.",
+    "weight": "80 g",
+    "volume": "250 ml",
+    "price": 200,
+    "price_postapoc": 20,
+    "material": [ "cotton" ],
+    "symbol": ",",
+    "color": "white",
+    "use_action": [ "WASH_HARD_ITEMS" ],
+    "snippet_category": [
+      { "id": "ktowel1", "text": "An elegant, monogrammed kitchen towel.  Unfortunately, it's not your initial." },
+      { "id": "ktowel2", "text": "An elegant, monogrammed kitchen towel.  It even has your initial!" },
```

```suggestion
      { "id": "ktowel1", "text": "An elegant, monogrammed kitchen towel.  Unfortunately, they aren't your initials." },
      { "id": "ktowel2", "text": "An elegant, monogrammed kitchen towel.  It even has your initials!" },
```
A "monogram" involves combining letters to make one thing ("mono" for one); unfortunately very confusable with "mono" being a single letter...